src/handlers/meta_transaction_handlers.ts

```diff
@@ -345,10 +360,40 @@ function parseV1RequestParams(req: express.Request): MetaTransactionV1QuoteReque
 
     // Parse sources
     let excludedSources: ERC20BridgeSource[], includedSources: ERC20BridgeSource[];
+    let rfqt: Pick<RfqRequestOpts, 'intentOnFilling' | 'isIndicative' | 'nativeExclusivelyRFQ'> | undefined;
+    const intentOnFilling = endpoint === 'quote' && txOrigin ? true : false;
     if (ENABLE_GASLESS_RFQT) {
         // if Gasless RFQt is enabled, parse RFQt params
-        // TODO: implement RFQt parsing logic
-        (excludedSources = []), (includedSources = []);
+        const parsedSourcesResult = parseUtils.parseRequestForExcludedSources(
+            {
+                excludedSources: req.query.excludedSources as string | undefined,
+                includedSources: req.query.includedSources as string | undefined,
+                intentOnFilling: intentOnFilling ? 'true' : 'false',
+                takerAddress,
+                txOrigin, // Gasless RFQt requires txOrigin in addition to takerAddress
+                apiKey,
```

Comment:
what is the apiKey here? Will it be a 0x internal one?  Since we do a lookup by integratorId, and that's currently passed in via a query parameter, would this apiKey just be used for securing down this endpoint (and not for integratorId lookup)?

Reply:
You're right in that the `apiKey` supposedly passed in here is not related to `integratorId`. If we were to make `apiKey` required for RFQt like vanilla `/swap`, then we can simply add extra validation to confirm that `apiKey` corresponds to the integrator, but the caller would need to supply both `apiKey` and `integratorId`. If we want to just use the `integratorId`, we would need to modify validation logic to bypass all `apiKey` checks.

Reply:
I think this is ok, but can we add a comment to highlight this behavior? Might be a little confusing since this logic drifts from `/swap` (and rightfully so)